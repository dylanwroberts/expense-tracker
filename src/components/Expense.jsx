import React from 'react';

export const Expense = props => (
    <div>
        <p>{props.expense.description}</p>
        <p>{props.expense.amount}</p>
    </div>
);

export default Expense;