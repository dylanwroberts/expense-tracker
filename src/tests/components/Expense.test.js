import React from 'react';
import { shallow } from 'enzyme';
import Expense from '../../components/Expense';
import expenses from '../fixtures/expenses';

test('should render an expense from test data', () => {
    const wrapper = shallow(<Expense expense={expenses[0]} />);

    expect(wrapper).toMatchSnapshot();
});