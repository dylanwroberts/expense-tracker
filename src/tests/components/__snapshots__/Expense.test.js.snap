// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an expense from test data 1`] = `
<div>
  <p>
    Food
  </p>
  <p>
    195
  </p>
</div>
`;
